pyatrea/__init__.py

```diff
@@ -82,328 +85,390 @@ def decompress(self, s):
-                            elif(title == '$ventilation'):
+                            elif title == "$ventilation":
                                 mode = AtreaMode.VENTILATION
-                            elif(title == '$circulation'):
+                            elif title == "$circulation":
                                 mode = AtreaMode.CIRCULATION
-                            elif(title == '$startUp'):
+                            elif title == "$startUp":
                                 mode = AtreaMode.STARTUP
-                            elif(title == '$runDown'):
+                            elif title == "$runDown":
                                 mode = AtreaMode.RUNDOWN
-                            elif(title == '$defrosting'):
+                            elif title == "$defrosting":
                                 mode = AtreaMode.DEFROSTING
-                            elif(title == '$external'):
+                            elif title == "$external":
                                 mode = AtreaMode.EXTERNAL
-                            elif(title == '$hpDefrosting'):
+                            elif title == "$hpDefrosting":
                                 mode = AtreaMode.HP_DEFROSTING
-                            elif(title == '$nightBefCool'):
+                            elif title == "$nightBefCool":
                                 mode = AtreaMode.NIGHT_PRECOOLING
-                            elif(title == 'IN1'):
+                            elif title == "IN1":
                                 mode = AtreaMode.IN1
-                            elif(title == 'IN2'):
+                            elif title == "IN2":
                                 mode = AtreaMode.IN2
-                            elif(title == 'D1'):
+                            elif title == "D1":
                                 mode = AtreaMode.D1
-                            elif(title == 'D2'):
+                            elif title == "D2":
                                 mode = AtreaMode.D2
-                            elif(title == 'D3'):
+                            elif title == "D3":
                                 mode = AtreaMode.D3
-                            elif(title == 'D4'):
+                            elif title == "D4":
                                 mode = AtreaMode.D4
-                            
-                            if(mode):
-                                self.modesToIds[mode] = int(option.attrib['id'])
-                                self.idsToModes[int(option.attrib['id'])] = mode
-                                if((not 'rw' in option.attrib) or option.attrib['rw'] == '1'):
+
+                            if mode:
+                                self.modesToIds[mode] = int(option.attrib["id"])
+                                self.idsToModes[int(option.attrib["id"])] = mode
+                                if (not "rw" in option.attrib) or option.attrib[
```

Comment:
Odd formating. Intentional?

Reply:
Yes, line shall not exceed certain number of characters... I will have to check for number and will add lint files to repo later.